runtime/message-channel.js

```diff
@@ -47,6 +47,8 @@ class MessageChannel {
     message = JSON.parse(JSON.stringify(message));
     if (this._ports[id]._onmessage) {
       try {
+        // Yield so that we deliver the message asynchronously.
```

Comment:
Is the need to do this `await 0` new to the use of js modules? Is it covered by a test or can we add one to make known use case explicit since it was surprising to me as a reader?

Reply:
This was unintentionally changed in https://github.com/PolymerLabs/arcs/pull/523 causing some tests to fail flakily for me.